esphome/components/bedjet/climate/bedjet_climate.cpp

```diff
@@ -236,9 +230,12 @@ void BedJetClimate::on_status(const BedjetStatusPacket *data) {
   if (converted_temp > 0)
     this->target_temperature = converted_temp;
 
-  converted_temp = bedjet_temp_to_c(data->ambient_temp_step);
-  if (converted_temp > 0)
+  converted_temp = bedjet_temp_to_c(this->temperature_source_ == TEMPERATURE_SOURCE_OUTLET    ? data->actual_temp_step
+                                    : this->temperature_source_ == TEMPERATURE_SOURCE_AMBIENT ? data->ambient_temp_step
+                                                                                              : 0);
```

Comment:
Multi level ternaries can get messy...
```suggestion
  if (this->temperature_source_ == TEMPERATURE_SOURCE_OUTLET) {
    converted_temp = bedjet_temp_to_c(data->actual_temp_step);
  } else {
    converted_temp = bedjet_temp_to_c(data->ambient_temp_step);
  }
```

Reply:
If we don't have the 3rd value, then it doesn't have to be multi-level:
```
  converted_temp = bedjet_temp_to_c(
    this->temperature_source_ == TEMPERATURE_SOURCE_OUTLET
      ? data->actual_temp_step
      : data->ambient_temp_step)
```
Since the source has to be one or the other, we don't have to fall back to 0

Reply:
If it has to even be multiline, it shouldnt be a ternary. The compiler will optimise it in the end, so I would rather it be easier to read.

Reply:
Ha. No strong feelings myself either way!

Change accepted, this is ready for review again.